tools/packaging/static-build/cache_components.sh

```diff
@@ -74,6 +74,33 @@ cache_virtiofsd_artifacts() {
 	create_cache_asset "${virtiofsd_tarball_name}" "${current_virtiofsd_version}" "${current_virtiofsd_image}"
 }
 
+cache_rootfs_artifacts() {
+	# We need to remove `-dirty` from teh osbuilder_last_commit as the rootfs artefacts are generated on that folder
```

Comment:
```suggestion
	# We need to remove `-dirty` from the osbuilder_last_commit as the rootfs artifacts are generated on that folder
```

Typo on `the`, and use US spelling instead of UK.